examples/mxnet/scenegraph/demo_reldn.py

```diff
@@ -0,0 +1,74 @@
+import dgl
+import argparse
+import mxnet as mx
+import gluoncv as gcv
+from gluoncv.data.transforms import presets
+from model import faster_rcnn_resnet101_v1d_custom, RelDN
+from utils import *
+from data import *
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Train Faster-RCNN networks e2e.')
```

Comment:
Is this a script just for detection or scene graph extraction? Also the Faster-RCNN net is not really trained right? Look like it is pre-trained instead.

Reply:
The Faster-CNN net needs to be trained. I could put the pretrained models to the same S3 bucket along with the dataset.